src/ci/resources/acquire_b2c_token.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script acquires a b2c token using gcloud auth and writes it to a file.
+# This file is used by renderCiResources to populate the CROMWELL_B2C_TOKEN in tes_application.conf.
+# It should be run as a "Before Launch" task in IntelliJ so the token is refreshed regularly.
+
+### Setting up this script for the first time in IntelliJ? 
+### 1. Open the "Run/Debug Configurations" dialog for the TES repo template.
+### 2. Add a new "Before Launch" task, selecting "Run External Tool". 
+###    This task should run BEFORE the renderCiResources task.
+### 3. Configure the new task:
+###   - Name: "Acquire b2c token"
+###   - Program: /full/path/to/src/ci/resources/acquire_b2c_token.sh
+###   - Arguments: ./env.temp
+###   - Working directory: /full/path/to/src/ci/resources
+###   - Advanced Options > Uncheck "Synchronize files after execution"
+### 4. Click "OK" to save the new task. Apply and close the "Run/Debug Configurations" dialog.
+
+# User must provide an output file path as an argument.
+# This can be provided by IntelliJ as a program argument in the run configuration.
+if [ -z "$1" ]; then
+    echo "Error: Must specify an output file path for the environment file."
+    exit 1
+fi
+
+# Acquire a b2c token using gcloud auth.
+echo "Using local gcloud auth to acquire a b2c token..."
+B2C_TOKEN=$(gcloud auth print-access-token)
```

Comment:
There is a tricky failure mode here. Sometimes `gcloud` will append a notice about taking a survey, or pending updates, to the token output. If you're feeling fancy you could try to detect this, it would be hard to debug why a token that we don't visually inspect is secretly invalid.

Reply:
Good call out! I changed the script slightly to redirect `stderr` into `dev/null` so these notices don't interfere with the acquired token.